### PR TITLE
Call newline conversion functions on installer output

### LINKED ISSUE #181: Bash installer is generated with CRLF linebreaks
As in title, the bash installer, for whatever reason, is generated with CRLF, which makes it effectively fail.

```
❯ cargo dist
analyzing workspace:
  portproxy
    [bin] portproxy

 WARN skipping powershell installer: not building any supported platforms (use --artifacts=global)
bundling portproxy-v0.2.0-aarch64-apple-darwin.tar.xz
bundling portproxy-v0.2.0-installer.sh
building cargo target (aarch64-apple-darwin/dist)
 (...)
bundled: /Users/mborejszo/source/portproxy/target/distrib/portproxy-v0.2.0-aarch64-apple-darwin.tar.xz
bundled: /Users/mborejszo/source/portproxy/target/distrib/portproxy-v0.2.0-installer.sh
announcing v0.2.0
  portproxy 0.2.0
    /Users/mborejszo/source/portproxy/target/distrib/portproxy-v0.2.0-installer.sh
    /Users/mborejszo/source/portproxy/target/distrib/portproxy-v0.2.0-aarch64-apple-darwin.tar.xz
      [bin] portproxy
      [misc] LICENSE.md, README.md
```

```
❯ file target/distrib/portproxy-v0.2.0-installer.sh
target/distrib/portproxy-v0.2.0-installer.sh: Bourne-Again shell script text executable, ASCII text, with CRLF, LF line terminators
```

```
❯ bash target/distrib/portproxy-v0.2.0-installer.sh
: command not foundproxy-v0.2.0-installer.sh: line 7:
: command not foundproxy-v0.2.0-installer.sh: line 14:
: invalid optionortproxy-v0.2.0-installer.sh: line 15: set: -
set: usage: set [--abefhkmnptuvxBCHP] [-o option] [arg ...]
: command not foundproxy-v0.2.0-installer.sh: line 16:
: command not foundproxy-v0.2.0-installer.sh: line 22:
'arget/distrib/portproxy-v0.2.0-installer.sh: line 23: syntax error near unexpected token `{
'arget/distrib/portproxy-v0.2.0-installer.sh: line 23: `download_binary_and_run_installer() {
```

Simple solution may be to run dos2unix conversion on the generated installer string.

### PR DESCRIPTION
**Pull request by spitfire05**
This fixes #181.

Also, makes sure the powershell script in CRLF convention, just for consistency.

### CODE REVIEW

**Review by ashleygwilliams — APPROVED** [review]
this looks good to me. i don't know much about this dependency- is it reliable? we could alternatively write the line substitution manually. curious what your experience has been with it.

i'm gonna write some integration tests per #188 and then rebase this on that and we should be good to merge. thank you so much for contributing!!

**Comment by spitfire05** [comment]
I'm not the best person to judge on this dependency, as I am the author of it. That being said, it is used in ~7 crates as dev-dependency for quite some time already.

Also, yes, a simple `string.replace` would be enough here as well probably (the `newline-converter` only upper hand is some small edge case scenario, and the fact that it is "already here"), but I'll leave it to the maintainers if this codebase goes more for declarativness or small dependency set :)

**Comment by ashleygwilliams** [comment]
ok, after messing with gitattributes for far too long, i'm gonna accept this and then also make a PR to add it to the CI yml generation and cut another pre-release

**Pull request merged**
